api/src/Serializer/Normalizer/RelatedCollectionLinkNormalizer.php

```diff
@@ -110,12 +113,15 @@ public function normalize($data, $format = null, array $context = []): null|arra
             if (isset($link['href'])) {
                 continue;
             }
-
-            try {
-                $normalized_data['_links'][$rel] = ['href' => $this->getRelatedCollectionHref($data, $rel, $context)];
-            } catch (UnsupportedRelationException $e) {
-                // The relation is not supported, or there is no matching filter defined on the related entity
-                continue;
+            // Only consider relations with non-null value (object or collection)
+            // (does not yet work, some properties are private and cannot be read out)
```

Comment:
Remove nonfunctional code before merging

Reply:
Agreed - if I can't get the idea to work, I'll remove the code again.

Reply:
solved